src/network/networkd-network-gperf.gperf

```diff
@@ -350,6 +350,7 @@ DHCPServer.BootServerAddress,                config_parse_in_addr_non_null,
 DHCPServer.BootServerName,                   config_parse_dns_name,                                    0,                             offsetof(Network, dhcp_server_boot_server_name)
 DHCPServer.BootFilename,                     config_parse_string,                                      CONFIG_PARSE_STRING_SAFE_AND_ASCII, offsetof(Network, dhcp_server_boot_filename)
 DHCPServer.RapidCommit,                      config_parse_bool,                                        0,                             offsetof(Network, dhcp_server_rapid_commit)
+SHCPServer.PersistLeases,                    config_parse_tristate,                                    0,                             offsetof(Network, dhcp_server_persist_leases)
```

Comment:
Looks like `networkd` just crashes
```
Failed to start systemd-networkd.service.
systemd-networkd.service: Scheduled restart job, restart counter is at 4.
Starting systemd-networkd.service...
/etc/systemd/networkd.conf:36: Unknown section 'DHCPServer'. Ignoring.
lo: Link UP
lo: Gained carrier
host0: Configuring with /usr/lib/systemd/network/80-container-host0.network.
free(): invalid pointer
systemd-networkd.service: Main process exited, code=dumped, status=6/ABRT
systemd-networkd.service: Failed with result 'core-dump'.
Failed to start systemd-networkd.service.
systemd-networkd.service: Scheduled restart job, restart counter is at 5.
systemd-networkd.service: Start request repeated too quickly.
systemd-networkd.service: Failed with result 'core-dump'.
Failed to start systemd-networkd.service.
```
I'm not sure why the CI didn't catch this.

Reply:
Ouch. Will fix soon.

Reply:
-> #31994.